dashboard/test/controllers/api/v1/peer_review_submissions_controller_test.rb

```diff
@@ -101,6 +102,38 @@ class Api::V1::PeerReviewSubmissionsControllerTest < ActionController::TestCase
     )
   end
 
+  test 'Peer Review report returns expected columns' do
+    create :peer_review, reviewer: @submitter, script: @course_unit.script
+    create :peer_review, reviewer: @submitter
+
+    get :report, params: {course_id: @course_unit.plc_course_id}
+
+    assert_response :success
+    response = CSV.parse(@response.body)
+
+    expected_headers = ['Name']
+    expected_headers << [@level_1, @level_2, @level_3, @level_4].map do |level|
+      [level.name.titleize, "#{level.name.titleize} Submit Date"]
+    end
+    expected_headers << 'Reviews Performed'
+
+    date = Time.now.strftime("%-m/%-d/%Y")
+
+    assert_equal [
+      expected_headers.flatten,
+      [@submitter.name, 'Unsubmitted', date, 'Unsubmitted', date, 'Accepted', date, 'Unsubmitted', date, '1']
+    ], response
+  end
+
+  [:teacher, :facilitator, :student].each do |user|
+    test_user_gets_response_for(
+      :report,
+      params: {course_id: 1},
+      user: user,
+      response: :forbidden
+    )
+  end
```

Comment:
why are these forbidden? There's no logged in user in the other test case either, is there? Please add a comment.

Reply:
There is -  if you look at the setup method, we are calling `sign_in(@plc_reviewer)`

Reply:
Ah, that makes sense. I missed it in setup 👍